lib/rubocop/ast/node/if_node.rb

```diff
@@ -98,6 +98,13 @@ def nested_conditional?
         false
       end
 
+      # Returns the condition of the `if` node.
+      #
+      # @return [Node] the condition of the `if` node.
+      def condition
```

Comment:
This is already defined in the `ConditionalNode` mixin. Please don't override it. 😊